### PR TITLE
[PPML] Upgrade Log4j In Trusted-big-data-scala-occlum's Dockerfile

### PR DESCRIPTION
**Pull request by piaolaidelangman**
* Delete `slf4j-log4j12-1.7.30.jar`.
* Add `log4j2.xml`.

### CODE REVIEW

**Review by qiyuangong — APPROVED** [review]
LGTM

File: ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile [hunk 1]
```diff
@@ -41,7 +41,8 @@ RUN cd /opt && \
     rm spark/jars/spark-kubernetes_2.12-$SPARK_VERSION.jar && \
     rm spark/jars/spark-network-common_2.12-$SPARK_VERSION.jar && \
     rm spark/jars/hadoop-common-3.2.0.jar && \
-    rm spark/jars/hive-exec-2.3.7-core.jar
+    rm spark/jars/hive-exec-2.3.7-core.jar && \
```
Comment (glorysdj) [inline ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile:44]: and /opt/spark/jars/log4j-1.2.17.jar?
and need replacements

Comment (piaolaidelangman) [reply ppml/trusted-big-data-ml/scala/docker-occlum/Dockerfile:44]: Updated.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**